drivers/xen/regions.c

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/xen/generic.h>
```

Comment:
Do you follow any rules to order includes?

Reply:
fixed